src/util/hass-util.html

```diff
@@ -63,12 +65,23 @@
   Object.keys(attributes).forEach(function (key) {
     customEl[key] = attributes[key];
   });
+  customEl.isVisible = true;
```

Comment:
I could add 'isVisible':false to stateObj before calling dynamicContentUpdater in more-info-content.html

What do you think?